app/src/test/java/org/dhis2/usescases/enrollment/EnrollmentPresenterImplTest.kt

```diff
@@ -68,30 +68,6 @@ class EnrollmentPresenterImplTest {
         )
     }
 
```

Comment:
maybe you coud add a unit test here that check if you can navigate back for each removed test